universal-application-tool-0.0.1/app/modules/SecurityModule.java

```diff
@@ -156,17 +157,51 @@ protected OidcClient provideAdClient(
       return null;
     }
     OidcConfiguration config = new OidcConfiguration();
+    // this is the resource identifier that tells adfs that this is civiform
```

Comment:
We're following the Google java style guide. So comments  have Initial caps, trailing periods etc.

ADFS should be capitalized too I believe.

Reply:
edited a lil and tried to use AD since it's active directory not adfs or azure ad